packages/web-components/src/components/ic-button/ic-button.css

```diff
@@ -540,6 +540,7 @@
   height: var(--height, 2.5rem);
   width: 2.5rem;
   padding: var(--ic-space-xs);
+  display: -webkit-box; /* in safari, this will render the icon on smaller devices */
```

Comment:
should this also be added to the other sizes? i.e. button-size-small & .button-size-default ?

Reply:
I think that we can add this later to the other buttons if an issue is opened for it. Easy to add later if needed.

Reply:
If that's ok for you guys, I'll reopen a new PR with the changes.

Reply:
yep, a new PR is fine. Think it would be good to fix all button sizes at once though. Whilst not visible on the site, there could be issues for others using those sizes